### PR TITLE
Add recensio controlpanel.

### PR DESCRIPTION
**Pull request by thet**

### CODE REVIEW

File: src/recensio/plone/browser/configure.zcml [hunk 1]
```diff
@@ -9,6 +9,13 @@
       layer="recensio.plone.interfaces.IRecensioPloneLayer"
       />
 
+  <!-- Control panel -->
+  <browser:page
+      name="recensio-settings"
+      for="Products.CMFPlone.interfaces.IPloneSiteRoot"
+      class=".controlpanel.RecensioSettingsControlPanel"
+      permission="cmf.ManagePortal" />
```
Comment (ale-rt) [inline src/recensio/plone/browser/configure.zcml:17]: I would add a layer attribute

Comment (ale-rt) [reply src/recensio/plone/browser/configure.zcml:17]: I would also move this to a `controlpanel` folder.
The `browser` folder will get pretty crowded.

File: src/recensio/plone/interfaces.py [hunk 2]
```diff
@@ -8,3 +10,153 @@ class IRecensioPloneLayer(IDefaultBrowserLayer):
 
 class IReview(Interface):
     """Marker interface for reviews and other select content types."""
+
+
+class IRecensioSettings(Interface):
```
Comment (ale-rt) [inline src/recensio/plone/interfaces.py:15]: Please keep this in the controlpanel folder

File: src/recensio/plone/profiles/default/registry/settings.xml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<registry>
+  <records interface="recensio.plone.interfaces.IRecensioSettings" />
```
Comment (ale-rt) [inline src/recensio/plone/profiles/default/registry/settings.xml:3]: please add a prefix like `prefix="recensio.plone.settings"`

Comment (thet) [reply src/recensio/plone/profiles/default/registry/settings.xml]: did it and I think it's a good idea giving us clean namespaces. however, settings now need always the prefix `recensio.plone.settings`.

Comment (ale-rt) [reply src/recensio/plone/profiles/default/registry/settings.xml]: Sorry, can you explain?

Comment (thet) [reply src/recensio/plone/profiles/default/registry/settings.xml]: in the old code we do not have this prefix. now we need to add it whereever we use a setting. You can see this in this commit: https://github.com/syslabcom/recensio.plone/pull/62/commits/b8843a767d39340df00e5e2085221f27a649e5b5

but it's not a big deal, I'm fine with it.

File: src/recensio/plone/browser/review.py [hunk 4]
```diff
@@ -39,7 +39,7 @@ def get_canonical_url(self):
         """XXX.
 
         registry = getUtility(IRegistry)
-        recensio_settings = registry.forInterface(IRecensioSettings)
+        recensio_settings = registry.forInterface(IRecensioSettings, prefix="recensio.plone.settings")
```
Comment (ale-rt) [inline src/recensio/plone/browser/review.py:42]: I think I get you comment about the prefix.
This is not a big deal once you start using plone.api, e.g.:

```
api.portal.get_registry_record("recensio.plone.settings.external_portal_url")
```

**Review by ale-rt — COMMENTED** [review]
LGTM

**Review by ale-rt — APPROVED** [review]

File: src/recensio/plone/profiles/default/registry/settings.xml (hunk 3 above)
Comment (ale-rt) [reply src/recensio/plone/profiles/default/registry/settings.xml]: Uhm, I see your point and I am thinking about removing the prefix now...

**Pull request merged**